test/testdata/autogen/generator.rb.autogen.exp

```diff
@@ -36,33 +36,33 @@ requires: []
  loc=test/testdata/autogen/generator.rb:4
  is_defining_ref=0
 [ref id=2]
+ scope=[Gener]
+ name=[HydraResource]
+ nesting=[[Gener]]
+ resolved=[Gener HydraResource]
+ loc=test/testdata/autogen/generator.rb:10
+ is_defining_ref=0
+[ref id=3]
  scope=[Gener]
  name=[HydraResource]
  nesting=[[Gener]]
  resolved=[Gener HydraResource]
  loc=test/testdata/autogen/generator.rb:6
  is_defining_ref=1
-[ref id=3]
+[ref id=4]
  scope=[Gener]
  name=[T]
  nesting=[[Gener]]
  resolved=[T]
  loc=test/testdata/autogen/generator.rb:6
  is_defining_ref=0
-[ref id=4]
+[ref id=5]
  scope=[Gener]
  name=[DOES_NOT_EXIST]
  nesting=[[Gener]]
  resolved=[]
  loc=test/testdata/autogen/generator.rb:6
  is_defining_ref=0
-[ref id=5]
- scope=[Gener]
- name=[HydraResource]
- nesting=[[Gener]]
- resolved=[Gener HydraResource]
- loc=test/testdata/autogen/generator.rb:10
- is_defining_ref=0
```

Comment:
Ah, sorry, I forgot about this. There's a bug in that script that generates spurious edits to this test. Can you revert the changes in this file?

I've been meaning to fix this for a while and in the mean time I've just been suffering through the bad edits by manually fixing things.

Reply:
Done!